deps/build.jl

```diff
@@ -50,15 +50,17 @@ if libale_detected == false
             CreateDirectory(_libdir)
             @build_steps begin
                 ChangeDirectory(_srcdir)
-                `rm -rf Arcade-Learning-Environment-0.6.0`
+                `rm -rf Arcade-Learning-Environment-0.5.2`
+                `rm -rf v0.5.2.zip`
+				`rm -rf Arcade-Learning-Environment-0.6.0`
                 `rm -rf v0.6.0.zip`
-                `curl -LO https://github.com/mgbellemare/Arcade-Learning-Environment/archive/v0.6.0.zip`
+                `wget https://github.com/mgbellemare/Arcade-Learning-Environment/archive/v0.6.0.zip`
```

Comment:
I would use curl here, see e.g. [here](https://github.com/JuliaLang/julia/issues/374).

Reply:
Oh I see. I thought wget was shipped by default and curl had to be downloaded. Fixing it right away...

Reply:
@jbrea ✅ done